draft-ietf-webpush-encryption.md

```diff
@@ -201,6 +198,18 @@ Note that a push service is not required to support more than 4096 octets of
 payload body, which equates to 4080 octets of cleartext, so the `rs` parameter
 can be omitted for messages that fit within this limit.
 
+An application server MUST NOT use other content encodings for push messages.
+In particular, content encodings that compress could result in leaking of push
+message contents.  The Content-Encoding header field therefore has exactly one
+value, which is `aesgcm128`.
```

Comment:
The "exactly one value" line linguistically confirms this, but since the beginning of this paragraph focuses on _other_ content encodings rather than _repeated_ content encodings, we disallow multiple rounds of aesgcm128 as well, right?

(HTTP encryption coding has an example that demonstrates this: https://tools.ietf.org/html/draft-thomson-http-encryption-01#section-5.3)